db/migrations/20221226113501_users.sql

```diff
@@ -0,0 +1,19 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT,
+  email VARCHAR(50) NOT NULL ,
+  kakao_id BIGINT(30) NULL,
+  name VARCHAR(10) NOT NULL,
+  profile_image VARCHAR(100) NULL,
+  phone_num VARCHAR(11) NULL,
+  birth_date VARCHAR(15) NULL,
+  password BINARY(60) NULL,
+  user_type_id INT NOT NULL DEFAULT 1,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY(id),
+  FOREIGN KEY (user_type_id) REFERENCES user_types(id)
+);
```

Comment:
- email, kakao_id는 unique키로 관리해주세요!
- 카카오톡에서 nickname을 설정하지 않으면 nickname이 `null`로 들어올 수도 있습니다. 해당 부분 참고하여 수정해주세요.
- phone_num 보단 `phone_number`로 구체적인 컬럼명을 사용해주세요. 
- birth_date도 좋지만 birthdate로 붙여서 사용해도 괜찮을 것 같습니다.
- 추가로 추후 외부 결제 API 사용 여부에 따라 POINT 컬럼 추가 시 pr 올려주시면 바로 반영해드리겠습니다~!

```suggestion
CREATE TABLE users (
  id INT NOT NULL AUTO_INCREMENT,
  email VARCHAR(255) NULL ,
  kakao_id BIGINT NULL,
  name VARCHAR(20) NULL,
  profile_image VARCHAR(2000) NULL,
  phone_number VARCHAR(11) NULL,
  birthdate VARCHAR(15) NULL,
  password BINARY(60) NULL,
  user_type_id INT NOT NULL DEFAULT 1,
  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY(id),
  FOREIGN KEY (user_type_id) REFERENCES user_types(id),
  CONSTRIANT users_ukey UNIQUE(email, kakao_id);
);
```